### PR TITLE
[AAP-29480] Remove old Event Streams code. Rename webhooks to event streams

### PR DESCRIPTION
**Pull request by lgalis**
Remove old Event Streams code. 
Rename Webhooks to Event Streams

### CODE REVIEW

File: frontend/eda/event-streams/EventStreams.tsx [hunk 1]
```diff
@@ -24,16 +24,22 @@ export function EventStreams() {
...
-      <PageHeader title={t('Event Streams')} description={t('Event streams.')} />
+      <PageHeader
+        title={t('Event Streams')}
+        description={t(
+          'Event streams represent server side webhooks which ease the routing issues related to running webhooks ' +
+            'individually in a container or a pod. Sources can be swapped in a rulebook with a matching event stream.'
+        )}
+      />
       <PageTable
-        id="eda-event-streams-table"
```
Comment (appuk) [inline frontend/eda/event-streams/EventStreams.tsx:29]: the id here should be `eda-event-streams-table` instead of `eda-webhooks-table` ??

**Review by appuk — APPROVED** [review]

File: frontend/eda/event-streams/EventStreams.tsx (hunk 1 above)
Comment (lgalis) [reply frontend/eda/event-streams/EventStreams.tsx:29]: Will change it in a follow up PR

File: frontend/eda/event-streams/components/PageFormEventStreamsSelect.tsx [hunk 2]
```diff
@@ -1,30 +1,30 @@
 import { FieldPath, FieldValues } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 import { PageFormMultiInput } from '../../../../framework/PageForm/Inputs/PageFormMultiInput';
-import { EdaWebhook } from '../../interfaces/EdaWebhook';
-import { useSelectWebhooks } from '../hooks/useSelectWebhooks';
+import { EdaEventStream } from '../../interfaces/EdaEventStream';
+import { useSelectEventStreams } from '../hooks/useSelectEventStreams';
 
-export function PageFormWebhookSelect<
+export function PageFormEventStreamSelect<
```
Comment (marshmalien) [inline frontend/eda/event-streams/components/PageFormEventStreamsSelect.tsx:7]: We can remove `PageFormEventStreamSelect` in a follow-up pr

File: frontend/eda/event-streams/components/EdaEventStreamCell.tsx [hunk 3]
```diff
@@ -1,28 +1,28 @@
 import { TextCell, useGetPageUrl } from '../../../../framework';
 import { useGet } from '../../../common/crud/useGet';
-import { EdaWebhook } from '../../interfaces/EdaWebhook';
+import { EdaEventStream } from '../../interfaces/EdaEventStream';
 import { EdaRoute } from '../../main/EdaRoutes';
 
-export function EdaWebhookCell(props: { webhook_id?: number | null }) {
+export function EdaEventStreamCell(props: { event_stream_id?: number | null }) {
```
Comment (marshmalien) [inline frontend/eda/event-streams/components/EdaEventStreamCell.tsx:6]: We can remove `EdaEventStreamCell` in a follow-up pr

File: frontend/eda/event-streams/hooks/useSelectEventStreams.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { edaAPI } from '../../../eda/common/eda-utils';
+import { useEventStreamFilters } from './useEventStreamFilters';
+import { useEventStreamColumns } from './useEventStreamColumns';
+import { EdaEventStream } from '../../interfaces/EdaEventStream';
+import { useEdaView } from '../../common/useEventDrivenView';
+import { MultiSelectDialog, usePageDialog } from '../../../../framework';
+
+export function useSelectEventStreams(eventStreamType?: number, title?: string) {
```
Comment (marshmalien) [inline frontend/eda/event-streams/hooks/useSelectEventStreams.tsx:10]: If we remove `PageFormEventStreamSelect`, we may be able to remove `useSelectEventStreams` and `SelectEdaEventStreams` too

File: frontend/eda/event-streams/EventStreams.cy.tsx [hunk 5]
```diff
@@ -9,116 +9,235 @@ describe('EventStreams.cy.ts', () => {
...
+            events_received: 0,
+            last_event_received_at: null,
           },
         ],
       }
     );
   });
 
-  it('Renders the correct event streams columns', () => {
+  it('Renders the correct webhooks columns', () => {
```
Comment (marshmalien) [inline frontend/eda/event-streams/EventStreams.cy.tsx:206]: Should `webhooks` be `event streams`?

**Pull request merged**